src/common/io.ts

```diff
@@ -1010,6 +1001,7 @@ export class IOWriteable extends IOReadable implements AoIOWrite {
   async decreaseOperatorStake(
     params: {
       decreaseQty: number | mIOToken;
+      instant?: boolean;
```

Comment:
we have instant withdrawal support availble for Decrease-Opertor-stake on contract